src/mixins/I13nMixin.js

```diff
@@ -148,8 +157,87 @@ var I13nMixin = {
         if (IS_DEBUG_MODE) {
             this._debugDashboard.destroy();
         }
+
+        this._removeSubComponentsListenersAndDebugDashboards();
+    },
+  
+    /**
+     * scan links
+     * @method _scanLinks
+     * @private
+     */
+    _scanLinks: function () {
+        var self = this;
+        var DOMNode = self.getDOMNode();
+        var foundElements = [];
+        var reactI13n = self._getReactI13n();
+        var scanTags = (self.props.scanLinks && self.props.scanLinks.tags) || DEFAULT_SCAN_TAGS;
+        if (!DOMNode) {
+            return;
+        }
+        self._subI13nComponents = [];
+
+        // find all links
+        scanTags.forEach(function scanElements(tagName) {
+            var collections = DOMNode.getElementsByTagName(tagName);
+            if (collections) {
+                foundElements = foundElements.concat(Array.prototype.slice.call(collections));
+            }
+        })
+
+        // for each link
+        // 1. create a i13n node
+        // 2. bind the click event
+        // 3. fire created event
+        foundElements.forEach(function registerFoundElement(element) {
+            var I13nNode = reactI13n.getI13nNodeClass();
+            var i13nNode = new I13nNode(self._i13nNode, {}, true, reactI13n.isViewportEnabled());
```

Comment:
Just want to make sure I understand how these i13nNode objects are destroyed.  I will in componentWillUnmount, the i13nNode created for the component (with the I13nMixin) is removed from its parent.  So I guess that is enough, because the subcomponent's i13n nodes will be detached and destroyed?